enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/subscription/SubscriptionDataStoreImpl.java

```diff
@@ -111,7 +112,7 @@ public API getApiByContextAndVersion(String uuid) {
 
     @Override
     public SubscriptionPolicy getSubscriptionPolicyByOrgIdAndName(String orgId, String policyName) {
-        String organizationId = StringUtils.isEmpty(orgId) ? APIConstants.SUPER_TENANT_DOMAIN_NAME : orgId;
+        String organizationId = FeatureFlags.getCustomSubscriptionPolicyHandlingOrg(orgId);
```

Comment:
@VirajSalaka Please check this considering the reason you mentioned in the chat..

Reply:
@slahirucd7 the reason is,

```java
SubscriptionPolicy subPolicy = datastore.getSubscriptionPolicyByOrgIdAndName(apiConfig.getOrganizationId(),
                sub.getPolicyId());
```

We are getting org ID from API. API always has an org. So let's say org Foo's API is invoked and his org is not in feature flag.

```java
    @Override
    public SubscriptionPolicy getSubscriptionPolicyByOrgIdAndName(String orgId, String policyName) {
        String organizationId = StringUtils.isEmpty(orgId) ? APIConstants.SUPER_TENANT_DOMAIN_NAME : orgId;
        String key = PolicyType.SUBSCRIPTION +
                SubscriptionDataStoreUtil.DELEM_PERIOD + organizationId +
                SubscriptionDataStoreUtil.DELEM_PERIOD + policyName;
        return subscriptionPolicyMap.get(key);
    }
```

But every org still does not have a sub policy right? until we are doing the migration.

Reply:
After merging the interceptor PR API invoking ORGs will have the default sub policies. I merged it now.